easybuild/easyconfigs/j/jupyterlab/jupyterlab-0.32.2-CrayGNU-18.08.eb

```diff
@@ -0,0 +1,75 @@
+# @author: robinson, sarafael 
+
+easyblock = 'Bundle'
+
+name = 'jupyterlab'
+version = '0.35.2'
+
+py_maj_ver = '3'
+py_min_ver = '6'
+py_rev_ver = '5.1'
+
+pyver = '%s.%s.%s' % (py_maj_ver, py_min_ver, py_rev_ver)
+pyshortver = '%s.%s' % (py_maj_ver, py_min_ver)
+
+homepage = 'https://github.com/jupyterlab/jupyterlab'
+description = "An extensible environment for interactive and reproducible computing, based on the Jupyter Notebook and Architecture."
+
+toolchain = {'name': 'CrayGNU', 'version': '18.08'}
+toolchainopts = {'pic': True, 'verbose': False}
+
+dependencies = [
+    ('jupyter', '1.0.0'),
+    ('jupyterhub', '0.9.4')
+]
+
+# bundle of Python packages
+exts_defaultclass = 'PythonPackage'
+
+exts_list = [
+    (name, version, {
+        'req_py_majver': '3',
+        'req_py_minver': '6',
+        'use_pip': True,
+        'source_urls': ['https://pypi.python.org/packages/source/j/jupyterlab/'],
+        }),
+    ('jupyterlab_server', '0.2.0', {
+        'req_py_majver': '3',
+        'req_py_minver': '6',
+        'use_pip': True,
+        'source_urls': ['https://pypi.python.org/packages/source/j/jupyterlab_server/'],
+        }),
+    ('jupyterlab_launcher', '0.10.0', {
+        'req_py_majver': '3',
+        'req_py_minver': '6',
+        'use_pip': True,
+        'source_urls': ['https://pypi.python.org/packages/source/j/jupyterlab_launcher/'],
+        }),
+]
+
+# specify that Bundle easyblock should run a full sanity check, rather than just trying to load the module
+full_sanity_check = True
+
+modextrapaths = {'PYTHONPATH': ['lib/python%s/site-packages' % pyshortver]}
+
+modextravars = {
+    'JUPYTERLAB_DIR': "%(installdir)s/share/jupyter/lab/",
+}
+
+# install labhub extension
+postinstallcmds = ["export JUPYTERLAB_DIR=%(installdir)s/share/jupyter/lab/; "
+                   "export PYTHONPATH=%(installdir)s/lib/python3.6/site-packages:$PYTHONPATH; "
```

Comment:
We could add here `"export YARN_CACHE_FOLDER=/tmp/$USER/yarn_cache; "`, to avoid creating the cache on `$HOME/.cache`. What do you think?

Reply:
Yes, you are very right!